### PR TITLE
QR Code Auth: Part 2 - Layouts, styles, strings, and illustrations

### PR DESCRIPTION
**Pull request by zwarm**
Parent #16481 

This PR adds the following for the QRCodeAuth flow
- Layout files for loading, content, and error views
- Update qrcodeauth_fragment to include the above layouts files
- Adds qrcodeauth_styles
- Adds string resources

Notes: 
- This PR will be merged to a feature branch and not trunk. Milestone will be set to "future"
- Not all strings declared are used in this PR, but they will be before the feature branch is requested to be merged in to trunk.

**Merge Instructions**
- Ensure that #16706 has been merged to the feature branch
- Merge as normal

**To test:**
- Install the app
- Login using a WP.com account
- Navigate to Me -> App Settings -> Debug Settings
- Enable `QRCodeAuthFlowFeatureConfig`
- Restart the App
- Navigate back to Me
-Tap the "Scan Login Code" row item
- The view should scroll through the above layouts - (You will not see the entire view, just the progress indicator and the buttons)

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [X] I have completed the Regression Notes.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        tools:src="@drawable/img_illustration_empty_results_216dp" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/error_title"
+        style="@style/QRCodeAuth.Title"
+        app:layout_constraintBottom_toTopOf="@+id/error_subtitle"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/error_image"
+        tools:text="No internet connection" />
```
Comment (ashiagr) [inline WordPress/src/main/res/layout/qrcodeauth_error_view.xml:30]: Np: This hardcoded text can be replaced with `qrcode_auth_flow_error_no_connection_title` in `strings.xml`.

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/error_subtitle"
+        style="@style/QRCodeAuth.Subtitle"
+        app:layout_constraintBottom_toTopOf="@+id/error_primary_action"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/error_title"
+        tools:text="Need an internet connection to scan a qr code" />
```
Comment (ashiagr) [inline WordPress/src/main/res/layout/qrcodeauth_error_view.xml:39]: Np: This hardcoded text can be replaced with `qrcode_auth_flow_error_no_connection_subtitle` in `strings.xml`.

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/error_container"
```
Comment (ashiagr) [inline WordPress/src/main/res/layout/qrcodeauth_error_view.xml:5]: I'm aware that UI is not ready for the flow. We might want to reuse `ActionableEmptyView` for the error layout if we can eliminate the cancel button and use a cross/ back button in a toolbar. Let's just keep it in the back of our minds for the future.

**Review by ashiagr — APPROVED** [review]
Looks good! Nice to see the layouts on a device with the temporary logic that you added. 🙂
I left minor nitpicks, you can merge the PR once those are addressed.

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml (hunk 3 above)
Comment (zwarm) [reply WordPress/src/main/res/layout/qrcodeauth_error_view.xml:5]: @ashiagr - Funny that you say that, I had every intention of using ActionableEmptyView, but I need the "cancel" link and not the back button to match the iOS design. I "borrowed" AEV and adjusted it for this use case. 👍

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml (hunk 1 above)
Comment (zwarm) [reply WordPress/src/main/res/layout/qrcodeauth_error_view.xml:30]: I will make this change in the final PR, so I don't have to go back and update the remaining 5 open PRs. Thank you. 🙇

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml (hunk 2 above)
Comment (zwarm) [reply WordPress/src/main/res/layout/qrcodeauth_error_view.xml:39]: I will make this change in the final PR, so I don't have to go back and update the remaining 5 open PRs. Thank you. 🙇

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml (hunk 1 above)
Comment (zwarm) [reply WordPress/src/main/res/layout/qrcodeauth_error_view.xml:30]: See [a1a1083](https://github.com/wordpress-mobile/WordPress-Android/pull/16713/commits/a1a1083ce6c007dcff5e6d4f2fa3c218a5757f37)

File: WordPress/src/main/res/layout/qrcodeauth_error_view.xml (hunk 2 above)
Comment (zwarm) [reply WordPress/src/main/res/layout/qrcodeauth_error_view.xml:39]: See [a1a1083](https://github.com/wordpress-mobile/WordPress-Android/pull/16713/commits/a1a1083ce6c007dcff5e6d4f2fa3c218a5757f37)

**Pull request merged**